homeassistant/components/tolosauna/climate.py

```diff
@@ -0,0 +1,168 @@
+"""TOLO Sauna climate controls (main sauna control)."""
+
+from typing import Any, Optional
+
+from tololib.const import Calefaction
+
+from homeassistant.components.climate import (
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    ClimateEntity,
+)
+from homeassistant.components.climate.const import (
+    CURRENT_HVAC_DRY,
+    CURRENT_HVAC_HEAT,
+    CURRENT_HVAC_IDLE,
+    CURRENT_HVAC_OFF,
+    FAN_OFF,
+    FAN_ON,
+    HVAC_MODE_DRY,
+    SUPPORT_FAN_MODE,
+    SUPPORT_TARGET_HUMIDITY,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, PRECISION_WHOLE, TEMP_CELSIUS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import ToloSaunaCoordinatorEntity, ToloSaunaUpdateCoordinator
+from .const import (
+    DEFAULT_MAX_HUMIDITY,
+    DEFAULT_MAX_TEMP,
+    DEFAULT_MIN_HUMIDITY,
+    DEFAULT_MIN_TEMP,
+    DOMAIN,
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up climate controls for TOLO Sauna."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]["coordinator"]
+    await coordinator.async_config_entry_first_refresh()
```

Comment:
Move this to `__init__.py` if you plan to do more platforms from the same coordinator (assuming you will from your discord conversations)

Reply:
Where exacty, in which method? `async_setup_entry`?

Reply:
> Where exacty, in which method? `async_setup_entry`?

`async_setup_entry` in `__init__.py`

> Only line 46?

Yes

Reply:
@milanmeu thx for the quick reply, done.